### ISSUE TITLE
Add resolution threshold to correlate_dft_2d

### ISSUE THREAD

**Issue by rsanchezgarc** [teamtomo/libtilt#53]
correlate_dft_2d is quite sensitive to noise. It will benefit from skipping high frequencies if requested. Weighting frequencies could also be in scope

**Comment by alisterburt** [comment]
yep totally agree - this is true for projection/backprojection too and potentially a nice optimisation in these parts of the codebase. I hadn't thought about adding weights but that's a really nice idea
